src/main/g8/src/main/scala/$package$/DemoParallelism.scala

```diff
@@ -1,47 +1,45 @@
 package $package$
 
+import cats.effect.{ExitCode, IO}
 import com.typesafe.scalalogging.LazyLogging
-import monix.eval.Task
-import monix.execution.Scheduler
 
-import scala.concurrent.Await
-import scala.concurrent.duration._
+import scala.concurrent.duration.DurationInt
 import scala.util.Random
 
 object DemoParallelism extends LazyLogging {
 
   /**
-    * Demonstrates effective parallelism using Monix and Monte-Carlo pi calculation.
-    *
-    * @param iterations  Iterations to calculate pi for
-    * @param parallelism Concurrent processes
-    */
-  def apply(iterations: Long, parallelism: Int): Unit = {
+   * Demonstrates effective parallelism using Monix and Monte-Carlo pi calculation.
+   *
+   * @param iterations  Iterations to calculate pi for
+   * @param parallelism Concurrent processes
+   */
+  def apply(iterations: Long, parallelism: Int): IO[ExitCode] = {
     logger.info(f"Will approximate Pi with \$iterations%,d iterations of random bubble-filling and \$parallelism%,d parallel processes...")
 
-    // Scheduler for parallelism:
-    implicit val s: Scheduler = Scheduler.forkJoin(parallelism, maxThreads = 1000)
```

Comment:
You know, I'm kinda bummed out by missing out on the opportunity to showcase `cats.effect.Resource` like I did in https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/pull/4 . For a long time now I've felt like it is probably the most underutilized Datatype in all of Scala.